tarxx-example.cpp

```diff
@@ -26,13 +26,43 @@
 
 #include "tarxx.h"
 
-int main(const int argc, const char* const* const argv)
-{
+int tarFile(const int argc, const char* const* const argv) {
```

Comment:
suggest calling this `main_file` or `main_tar_file` or `main_file_mode`; at least for style consistency `tar_file`.

Reply:
it's `tar_streaming_mode` now